### ISSUE TITLE
How to do learning rate decay for tesseract-4?

### ISSUE THREAD

**Issue by shuguang101** [tesseract-ocr/tesseract#1704]
Is it possible to decay the learning rate  in LSTM training process by setting lstmtraining's parameter? I try using a new learning rate with the command lstmtraining  --continue_from some_check_point --learning_rate new_rate, but it dosen't work.

**Comment by zdenop** [comment]
Please respect guidelines for posting issue:  use tesseract user forum for asking question/support.
